packages/sms/src/commands/sms/index.ts

```diff
@@ -0,0 +1,60 @@
+import BaseCommand from '@vonage/cli-utils';
+import { request } from '@vonage/vetch'
```

Comment:
Hey @kellyjandrews I notice some lines end in ; others not. Maybe keep it consistent across the files?
```suggestion
import { request } from '@vonage/vetch';
```

Reply:
Need to get eslint set up properly so it will keep it consistent, yes.